raml-model/src/main/java/io/vrap/rmf/raml/model/RamlModelBuilder.java

```diff
@@ -107,11 +107,13 @@ public Boolean caseAnyType(final AnyType anyType) {
                 final EClass eClass = anyType.eClass();
                 if (eClass != superType.eClass()) {
                     final AnyType newInlineType = (AnyType) EcoreUtil.create(superType.eClass());
-                    for (EAttribute attribute : eClass.getEAllAttributes()) {
-                        final Object value = anyType.eGet(attribute);
-                        newInlineType.eSet(attribute, value);
-                    }
-                    newInlineType.eSet(eClass.getEStructuralFeature("type"), superType);
+                    final String name = anyType.getName() == null ? superType.getName() : anyType.getName();
+                    newInlineType.setName(name);
```

Comment:
Instead of dynamically setting the properties I switched to use the type specific api. This is required because some of our properties (e.g. description) aren't attributes anymore and would have been lost with the previous dynamic code.